qa/tasks/cephfs/test_volumes.py

```diff
@@ -95,6 +95,27 @@ def tearDown(self):
             self._delete_test_volume()
         super(TestVolumes, self).tearDown()
 
+    def test_volume_rm(self):
+        try:
+            self._fs_cmd("volume", "rm", self.volname)
+        except CommandFailedError as ce:
+            if ce.exitstatus != errno.EPERM:
+                raise RuntimeError("expected the 'fs volume rm' command to fail with EPERM, " \
+                                   "but it failed with {0}".format(ce.exitstatus))
+            else:
+                try:
+                    self._fs_cmd("volume", "rm", self.volname, "--yes-i-really-mean-it")
+                except CommandFailedError as ce:
+                    #re-raise EPERM
```

Comment:
why do you need a `try catch` block here? You're just re-raising the exception anyway. You're not handling the exception in any special way.

Reply:
We expect the `volume rm <self.volname> --yes-i-really-mean-it` to succeed here. If it succeeds we don't re-raise. But if it fails, then it should mostly be a permission issue. May be I could give a check `if ce.exitstatus == errno.EPERM` then `raise`

Reply:
You don't need a `try catch` block here. If the command fails, let it fail.